litestar/utils/warnings.py

```diff
@@ -4,6 +4,13 @@
 from litestar.exceptions import LitestarWarning
 from litestar.types import AnyCallable, AnyGenerator
 
+__all__ = (
+    "warn_implicit_sync_to_thread",
+    "warn_pdb_on_exception",
+    "warn_sync_to_thread_with_async_callable",
+    "warn_sync_to_thread_with_generator",
+)
```

Comment:
idk if we want these exposed in documentation

Reply:
I decided to leave the flake8 dunder additions unmodified so those more experienced with the library can decide what to exclude 👍